src/commands/default/env.ts

```diff
@@ -46,7 +47,7 @@ export default class Peers extends BaseCommand<typeof Peers> {
       true,
     );
 
-    assert(envConfig !== undefined, "this command requires fluence project");
+    assert(envConfig !== null, "this command requires fluence project");
```

Comment:
sure there's no need to check for both null and undefined?

Reply:
with my ts super strict setup I can be completely sure about these things